src/services/atomic-swap.service.js

```diff
@@ -29,6 +29,17 @@ const {
  */
 const walletListenedProposals = new Map();
 
+/**
+ * Represents errors related to interactions with the Atomic Swap Service
+ */
+class SwapServiceError extends Error {
```

Comment:
The same comment as in the other PR.

Reply:
Added a dedicated `errors` file on [79a66a8](https://github.com/HathorNetwork/hathor-wallet-headless/pull/286/commits/79a66a8202d802f4edc0ccad3a2a7f395b43e5ec)